department-of-veterans-affairs/caseflow
# making label changes and fixing feature tests

Resolves [Intake Form: have inactive Appeals only show the 12 tasks](https://jira.devops.va.gov/browse/APPEALS-56221)

# Description
- Added seed data script to create inactive appeals tied to veterans 
> _Please Note: Seed data for inactive appeals will need to be added in demo before this will show as expected_
- Added the ability to find inactive appeals using the appeals search serializer
- Added the ability for inbound ops users to only see the 12 tasks relating to inactive appeals when adding a task related to an appeal
- Wrote tests to verify this functionality
- Addressed a misspelled banner and banner tests to match the AC of this story

## Acceptance Criteria

## Testing Plan

Go to [Jira Issue/Test Plan Link](https://jira.devops.va.gov/browse/JIRA-12345) or list them below
1. Log into Caseflow as an inbound ops team user
2. Navigate to queue/correspondence
3. Select a veteran from the unassigned tab to go to the review package page
4. If necessary, add a correspondence type, and click save.
5. Scroll to the bottom and select "Create record"
6. Navigate to step 2
7. Find the "Tasks related to an existing Appeal" section and click the "yes" radio button
8. Find an appeal where the "assigned to" is "Post-decision" (this will likely be on the last page), then click that checkbox.
9. Scroll down and click the "+ Add tasks" button
10. Click the task dropdown. 
11. Verify that there are only 12 options in this list. A quick way to check is to make sure the list does not contain "Change Of Address", as this is only available on active appeals
12. Scroll back up and choose another appeal that is not one of the last four and is not assigned to post-decision
13. For this appeal, select the "+ Add tasks" button and click the task dropdown. 
14. This time, the list should include more options and will now include "Change Of Address" in the list.
15. Functionality for each can be verified to make sure list options disappear when chosen (except for other motion, which should only disappear after 2 are chosen), and that a max of four tasks can be created by that user
